extensions/wikia/PortabilityDashboard/PortabilityDashboardModel.class.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+class PortabilityDashboardModel {
+	const PORTABILITY_DASHBOARD_TABLE = 'portability_dashboard';
+	const WIKI_ID_FIELD = 'wiki_id';
+	const CUSTOM_INFOBOXES_FIELD = 'custom_infoboxes';
+	const TYPELESS_FIELD = 'typeless';
+
+	private $connection;
+
+	const WIKIS_LIMIT = 500;
+
+	public function __construct( $db = null ) {
+		$this->connection = $db;
+	}
+
+	/**
+	 * gets model data for portability dashboard
+	 * @return array
+	 */
+	public function getList() {
+		$rowList = $this->getRowList();
+		$wikiParamsList = $this->getWikiParamsList( $rowList );
+
+		return $this->extendRowListWithWikiParams( $rowList, $wikiParamsList );
+	}
+
+	/**
+	 * gets row list from DB
+	 * @return bool|mixed
+	 */
+	private function getRowList() {
+		return ( new WikiaSQL() )
+			->SELECT_ALL()
+			->FROM( self::PORTABILITY_DASHBOARD_TABLE )
+			->ORDER_BY( 'migration_impact' )
+			->DESC()
+			->LIMIT( self::WIKIS_LIMIT )
+			->run( $this->readDB(), function ( ResultWrapper $rows ) {
+				$result = [ ];
+				while ( $row = $rows->fetchObject() ) {
+					$result[] = [
+						'wikiId' => $row->wiki_id,
+						'portability' => $row->portability,
+						'infoboxPortability' => $row->infobox_portability,
+						'traffic' => $row->traffic,
```

Comment:
The description states this is a % of overall Wikia traffic, whereas this is expressed in page view count